### DISCUSSION TITLE
How-To: Visualize UML diagrams in Eclipse

### DISCUSSION THREAD

**Discussion by javac9** [openmdx/openmdx-example#4] [Q&A] [answered]
I'm having issues in Eclipse by trying to visualize the existing UML models that come in openmdx-example projects (HelloWorld, Workshop). Which plugin do I actually need? Does anyone use that plugin with the latest version of Eclipse?

![Screenshot 2021-07-28 121948](https://user-images.githubusercontent.com/25393309/127306748-b2b0c75e-eb94-4963-b22f-4c0e70475426.jpg)

**Comment by wfro64** [comment, marked as answer]
You can visualize the models as follows:
* Install the Papyrus for Eclipse (https://www.eclipse.org/papyrus/index.php)
* Import the EMF and Papyrius projects into your workspace from ./openmdx-example/helloworld/src/model/
./openmdx-example/workshop/src/model/
* Opening models.di renders the diagrams

See screenshots:
![Screenshot_20210728_122844](https://user-images.githubusercontent.com/11387061/127308658-f863ea0c-50bd-437f-a0a6-a7a9fc88efeb.png)
![Screenshot_20210728_122856](https://user-images.githubusercontent.com/11387061/127308660-85f06103-57f2-458e-9e45-f30d643a7247.png)
![Screenshot_20210728_122911](https://user-images.githubusercontent.com/11387061/127308661-b1a421a5-b8a1-4473-878f-7e11ee2d5bdb.png)
